core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -401,6 +401,8 @@ class GroupCoordinator(val brokerId: Int,
             val memberMetadata = group.get(memberId)
             responseCallback(memberMetadata.assignment, Errors.NONE)
             completeAndScheduleNextHeartbeatExpiration(group, group.get(memberId))
+
+          case _ =>
```

Comment:
Don't need this because the `Dead` case is being handled already in the upstream.

Reply:
You mean, `if (group.is(Dead)) {` in the line 350, right? Then, how to suppress not-exhaustive match warning? Do you have any idea?

Reply:
We could raise an exception since we expect to have handled this state. I think it is better to be exhaustive with matches. In fact, I'd prefer to handle `case Dead` explicitly rather than a blind `case _`. That way the compiler will generate a warning if we add a new state and don't update this. In other words, the warning will have value.